test/integration/launchUtil.js

```diff
@@ -19,26 +19,48 @@ exports.launch = function (fixtureName, opts, done) {
     }
 
     var output = '';
-    lastLaunch = child_process.execFile(path.resolve(__dirname, opts.exec), [],
-        {
-            'cwd': path.resolve(__dirname, 'fixtures/' + fixtureName),
-            'env': opts.env
-        },
-        function (err, stdout, stderr) {
-            if (err) {
-                console.warn(err, stderr);
-            }
-            output += stdout + stderr;
-        });
+    var bosPath = path.resolve(__dirname, opts.exec);
+    if (process.platform === 'win32') {
+        lastLaunch = child_process.exec('node ' + bosPath,
```

Comment:
does `exec` work on *nix too?
I'm not sure we need this special case for 'win32', I think it will work on both